shared/src/business/useCases/createCaseFromPaperInteractor.js

```diff
@@ -124,6 +128,8 @@ exports.createCaseFromPaperInteractor = async (
 
   const petitionDocketEntryEntity = new DocketEntry(
     {
+      contactPrimary: caseToAdd.getContactPrimary(),
+      contactSecondary: caseToAdd.getContactSecondary(),
```

Comment:
No changes requested, but is this also an area where this sort of refactoring effort will need to be continued in the future?

Reply:
It seemed like it would be more difficult to use an array for the create case forms because only 2 petitioners are allowed in that instance, so we left it this way purposefully, unless there becomes a need to refactor further during the other stories.